system/Validation/Validation.php

```diff
@@ -790,7 +790,9 @@ protected function fillPlaceholders(array $rules, array $data): array
                         // Check if the validation rule for the placeholder exists
                         if ($placeholderRules === null) {
                             throw new LogicException(
-                                'No validation rules for the placeholder: ' . $field
+                                'No validation rules for the placeholder: "' . $field
+                                . '". You must set the validation rules for the field.'
+                                . ' See <https://codeigniter4.github.io/userguide/libraries/validation.html#validation-placeholders>.'
```

Comment:
```suggestion
                                . ' See <https://codeigniter.com/user_guide/libraries/validation.html#validation-placeholders>.'
```

Consistent link

Reply:
I would like to use it, but see https://codeigniter.com/user_guide/changelogs/index.html
Now the official user guide is not updated.
See https://github.com/codeigniter4/userguide/actions/runs/8027707768/job/21932010657